examples/webview-test-app/main.qml

```diff
@@ -0,0 +1,272 @@
+                    value: 5.0
+                    label: "Reload interval [s]"
+                    stepSize: 0.1
+                }
+
+                Item {
+                    width: parent.width
+                    height: Theme.paddingLarge
+                }
+
+                Button {
+                    text: mode.currentIndex === 0 ? "Open webpage" : "Start"
+                    anchors.horizontalCenter: parent.horizontalCenter
+
+                    onClicked: {
+                        page.startTest()
+                    }
+                }
+            }
+        }
+    }
+
+    Component {
+        id: webViewPageComponent
+
+        Page {
+            id: webViewPage
+
+            property int mode
+            property string webPageAddress
+            property real creationDelay
+            property real destructionDelay
+            property real reloadInterval
+
+            property bool webViewLoading: false
+            property int webViewLoadProgress: 0
+
+            anchors.fill: parent
+
+            showNavigationIndicator: false
+            backNavigation: false
+
+            Component.onCompleted: {
+                webViewLoader.setWebViewActive(true)
+            }
+
+            Timer {
+                id: webViewRecreationTimer
+
+                interval: (webViewLoader.active ?
+                               destructionDelay :
+                               creationDelay) * 1000.0
+
+                running: pageStack.currentPage === webViewPage && mode === 1
+                repeat: true
+
+                onTriggered: {
+                    webViewLoader.setWebViewActive(!webViewLoader.active)
+                    loadStatusShowTimer.start()
+                }
+            }
+
+            Component {
+                id: webViewComponent
+
+                WebView {
+                    id: webView
+
+                    anchors.fill: parent
+                    active: true
+
+                    Timer {
+                        id: webViewReloadTimer
+
+                        running: pageStack.currentPage === webViewPage && (mode === 2 || mode === 3)
+                        interval: reloadInterval * 1000.0
+                        repeat: true
+
+                        onTriggered: {
+                            console.log("webview reload")
+                            webView.reload()
+                        }
+                    }
+
+                    onLoadingChanged: {
+                        webViewPage.webViewLoading = loading
+                        webViewPage.webViewLoadProgress = 0
+                        loadStatusShowTimer.start()
+                    }
+
+                    onLoadProgressChanged: {
+                        webViewPage.webViewLoadProgress = loadProgress
+                    }
+                }
+            }
+
+            Loader {
+                id: webViewLoader
+
+                function setWebViewActive(value) {
```

Comment:
Is it not needed rather even a bit misleading. I'd rather just use ```webViewLoader.active``` where needed.

For logging purposes you could add

```
onActiveChanged: {
    if (value) {
        console.log("webview creation")
    } else {
        console.log("webview destruction")
    }
}
```

Changing a value of a property to the same that there was should not trigger change. That is the normal way how properties should be written in C++. This implies that...

```
webViewLoader.active = true
webViewLoader.active = true
```

Would log "webview creation" once given that previous active value was false.